feat(angular): Add documentation for Sentry's `ErrorHandler` (#11361)
getsentry/sentry-docs: platform-includes/getting-started-config/javascript.angular.mdx

export class AppModule {}
```

> The `Sentry.createErrorHandler` function initializes a Sentry-specific `ErrorHandler` that automatically sends errors caught by Angular to Sentry. Cou can further [configure](./features/error-handler) Sentry's `ErrorHandler` behavior.

Reviewer: ```suggestion
The `Sentry.createErrorHandler` function initializes a Sentry-specific `ErrorHandler` that automatically sends errors caught by Angular to Sentry. Learn how to further configure Sentry's `ErrorHandler` behavior [here](./features/error-handler).
```
